docs/setup.md

```diff
@@ -1,11 +1,28 @@
-| [Home](../README.md) |
-|--------------------------------------------|
-
 # Installation
```

Comment:
we can add a note saying Setup guide automatically installed as part of SFSP v3 and later.